index.html

```diff
@@ -1,30 +1,54 @@
-<head>
-  <link rel="stylesheet" href="sakura.css" type="text/css">
-  <link rel="stylesheet" href="sakura-dark.css" type="text/css">
-</head>
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>KISS - Community</title>
+        <meta charset="utf-8">
+        <meta name="description" content="Kiss linux community">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="./assets/css/normalize.css" rel="stylesheet">
+        <link href="./assets/css/skeleton.css" rel="stylesheet">
+        <link href="./assets/css/style.css" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap" rel="stylesheet">
+    </head>
 
-<body>
-  <h1> welcome to the <a href=https://kisslinux.org>kiss linux</a> community </h1>
+    <body>
+        <header>
+            <div class="container">
+                <h1>KISS Linux Community</h1>
+                <blockquote>when dylan says "no", we say "maybe"</blockquote>
+                <blockquote><a href="https://kisslinux.org/" target="_blank">KISS Linux</a> is an independent Linux(R) distribution with a focus on simplicity and the concept of less is more.
```

Comment:
Please remove `target="_blank"` in all `<a` tags - it changes browser defaults, and is nicer to be up to the user.

Reply:
done